plugins/qCompass/ccCompassInfo.h

```diff
@@ -7,88 +7,14 @@
 #include <qdialogbuttonbox.h>
 #include <qboxlayout.h>
 #include <qtextedit.h>
+#include <qfile.h>
```

Comment:
These should all use the same &lt;QFile&gt; &lt;QTextEdit&gt; style like you do with QTextStream, not the .h ones